src/System.Security.Cryptography.X509Certificates/ref/System.Security.Cryptography.X509Certificates.cs

```diff
@@ -9,13 +9,11 @@ namespace Microsoft.Win32.SafeHandles
 {
     public sealed partial class SafeX509ChainHandle : Microsoft.Win32.SafeHandles.SafeHandleZeroOrMinusOneIsInvalid
     {
-        internal SafeX509ChainHandle() : base(default(bool)) { }
-        public override bool IsInvalid { get { throw null; } }
```

Comment:
I assume the override was "removed" because the type is sealed (so there's no `base.IsInvalid` concern)?

Reply:
Ah.  That would be a good reason, too :smile:.

Looks like when we brought back SafeHandleZeroOrMinusOneIsInvalid the override was removed from the `src` copy but not the `ref`.